starter_code/src/movies.js

```diff
@@ -1,6 +1,19 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(duration) {
```

Comment:
This function should return an array, you need to return the result of the `movies.map` and not a duration